MIP/mip-21/README.md

```diff
@@ -0,0 +1,91 @@
+
+## Specification
+
+The Cross-Chain Transfer Protocol (CCTP) operates using a set of attestors that validate all bridge transactions occurring between chains. These attestors listen for bridge events, confirm the validity of transactions, and, if necessary, revert them by providing appropriate proofs. A transfer allowance managed by a centralized authority further ensures that breaches are mitigated by limiting the amount of value transferred.
+
+To align with this approach and reduce complexity:
+
+- **Bridge Relayer as Attestor**: The Bridge Relayer in our system will function as an attestor, operating via a multisig scheme (i.e. the attestor collects signatures from a threshold number of comittee members that approve of the message). This relayer, through aggregated signatures, produces a proof on a signed message.  
+
+- **User Interaction**: Users, or other parties interested in completing the bridge, retrieve their funds on the target chain by submitting the signature and message.
+
+- **Validation Mechanism**: The off-chain signature produced by the multisig scheme is verified against the message on-chain.
+The challenge is to make sure that the signature is only used once, but the beauty is that the signature is provided by a multi-party system that do not need to provide onchain signatures.
+Relying on a protocol like CCTP that has been tested in live environments significantly reduces the risk of any unexpected utilization of off-chain and on-chain proofs.  
+
+Or on Circle's own words:
+
+1. An on-chain component on source domain emits a message.
+2. Circle's off-chain attestation service signs the message.
+3. The on-chain component at the destination domain receives the message, and forwards the message body to the specified recipient.
+
+This approach minimizes the need for additional components and avoids the reliance on synthetic assets, leading to a simplified and cost-effective bridge solution.
+
+### Flow
+
+1. **User Deposits**:  
+   - The user deposits tokens on the source chain, which triggers a deposit event.
+   
+2. **Bridge Relayer**:  
+   - The Bridge Relayer, listening for the deposit event, picks up the message or messages associated with the deposit.
+   
+3. **Multisig Signing**:  
+   - The multisig party signs the bridge transaction. The combination of signatures produces an aggregated signature that ties the signatures to the message.
+
+4. **Target Chain Verification**:  
+   - On the target chain, the aggregated signature and the message are used to verify the validity of the bridge transaction. The withdrawal can only occur if the multisig-provided signature is valid.
```

Comment:
I would say that fast finality settlement is not scoped here. But, it can be done so.
First, I advocate for a multisig. This could be 5 keys of hours to prevent the bridge being hacked by stolen keys.
Second, it could be be part of the settlement party.
For the target chain if including FFS, on the layer 2, to learn about the stake, I would not be sure about that, Possibly what we could believe have is a service that adds and removes signers from multisigs on move that correspond to the attestors and their weights (eth is already covered with MCR staking).

Reply:
this makes sense. thx for clarifying